src/Microsoft.AspNet.OData.Shared/Formatter/Deserialization/DeserializationHelpers.cs

```diff
@@ -19,7 +23,54 @@ internal static class DeserializationHelpers
         internal static void ApplyProperty(ODataProperty property, IEdmStructuredTypeReference resourceType, object resource,
```

Comment:
>ApplyProperty [](start = 29, length = 13)

This provides case insensitivity only for payloads.  Isn't it a bit inconsistent to support case-insensitive properties in payloads, but not in URLs?

Reply:
I think case-insensitivity is already supported in WebAPI https://docs.microsoft.com/en-us/odata/webapi/custom-url-parsing

Reply:
@KenitoInc I think Mike's meaning is to support the value case-insensitive in the URL. @mikepizzo Is it correct? The Web API already supported case-insensitivity is for the key words in the request url.  For example:   $FilTer is same as $filter.

Reply:
By default, webapi supports case-insenstive values in the url. I can get results correctly from the 2 requests below.
`/Books?$expand=Authors&$filter=Id eq 1`
`/Books?$expand=authors&$filter=id eq 1`